### DISCUSSION TITLE
Hardware für Ethernet, CMT2300A und MCP2515

### DISCUSSION THREAD

**Discussion by Max8409** [hoylabs/OpenDTU-OnBattery#1073] [Q&A]
Hallo zusammen,
nachdem ich nun etliche Stunden recherchiert habe, bin ich langsam am Verzweifeln. Ich möchte eine DTU mit Ethernet Anschluss, dem CMT2300A Modul, der Möglichkeit das Huawei Netzteil einzubinden und die Kommunikation mit dem Pylontech CANbus. Abgesehen davon, dass ich trotz intensiver Suche keine geeignete Hardware gefunden habe, ist es derzeit ja wohl nicht möglich, das CMT Modul und das Huawei Netzteil gleichzeitig zu betreiben. Gibt es hier irgendeine Lösung bzw. ist abzusehen, ob / wann das in Zukunft möglich sein wird?
Vielen Dank und viele Grüße
Max

**Comment by AndreasBoehm** [comment]
Das Problem das CMT und Huawei nicht gleichzeitig genutzt werden können sollte bald gelöst sein: https://github.com/helgeerbe/OpenDTU-OnBattery/pull/1144
